tests/chain_tests/plugins/tags/tags_tests.cpp

```diff
@@ -296,7 +298,7 @@ SCORUM_TEST_CASE(check_pending_payout_after_first_payout)
 
     auto d = _api.get_content(post2.author(), post2.permlink());
 
-    BOOST_CHECK_EQUAL(d.pending_payout_scr, asset::from_string("0.000000005 SCR"));
+    BOOST_CHECK_EQUAL(d.pending_payout_scr, asset::from_string("0.000000009 SCR"));
```

Comment:
why payout is change?

Reply:
Because cash from advertising budgets is coming into balancer in different way now: rarely but bigger amounts. So it is because of the balancer